### ISSUE TITLE
Fix POM in `0.9.0`

### ISSUE THREAD

**Issue by mkobit** [mkobit/jenkins-pipeline-shared-libraries-gradle-plugin#86]
```
Starting a Gradle Daemon, 3 incompatible and 1 stopped Daemons could not be reused, use --status for details           

FAILURE: Build failed with an exception.

* What went wrong:                                         
A problem occurred configuring root project 'jenkins-pipeline-shared-library-example'.                                 
> Could not resolve all artifacts for configuration ':classpath'.                                                      
   > Could not find com.squareup.okhttp3:okhttp:.          
     Required by:                                          
         project : > com.mkobit.jenkins.pipelines.shared-library:com.mkobit.jenkins.pipelines.shared-library.gradle.plugin:0.9.0 > com.mkobit.jenkins.pipelines:jenkins-pipeline-shared-libraries-gradle-plugin:0.9.0     
```

Looks like a bug in Gradle

**Comment by Jonatha1983** [comment]
Exactly what I was about to write you , thanks !

**Comment by mkobit** [comment]
Plan is to reduce, reproduce, and report it to Gradle before fixing and releasing `0.9.1` today.

**Comment by Jonatha1983** [comment]
I think that part of the release flow should include the example repo upgrading and compiling with the new version , what do you think ?  would you like me to help in this  ?

**Comment by mkobit** [comment]
Reported upstream issue to Gradle at https://github.com/gradle/gradle/issues/9565

Regarding https://github.com/mkobit/jenkins-pipeline-shared-libraries-gradle-plugin/issues/86#issuecomment-496546913 , I think that is a great idea.

In the past I've manually done this by

* `./gradlew publishToMavenLocal` in this repository with some version
* Update the example to use `mavenLocal()` repository
* Update plugin version to match locally published one
* 
* Clean up local workspace to no longer use `mavenLocal()`
* Go forward with actual version publication

I didn't do it this time, otherwise I would have caught this issue before publishing. It is a bit manual, which is probably why I didn't do it :laughing: . I've thought about this a little before, and my hope would be that it is a part of the Circle CI pipeline to get automated feedback. I think some combination of composite builds with `--include-build` could be useful. I created https://github.com/mkobit/jenkins-pipeline-shared-libraries-gradle-plugin/issues/87 for tracking that.

**Comment by mkobit** [comment]
Released `0.9.1` and updated example in https://github.com/mkobit/jenkins-pipeline-shared-library-example/commit/0cfeda933e75d2dc7325ec93aa35bc59b6f181cd - closing
